packages/ui5-application-writer/test/index.test.ts

```diff
@@ -102,4 +108,16 @@ describe('UI5 templates', () => {
         fs.delete(join(projectDir, 'webapp/manifest.json'));
         await expect(enableTypescript(projectDir, fs)).rejects.toThrowError();
     });
+
+    it('Check that no ui5-local.yaml file is generated for CAP application', async () => {
+        const projectDir = join(outputDir, 'testapp-cap');
+        ui5AppConfig.app.projectType = 'CAPNodejs';
+        const fs = await generate(projectDir, { ...ui5AppConfig, ui5: { minUI5Version: '1.96.1' } });
+        // Check if ui5-local.yaml does not exist
+        expect(hasFileInFsDump(fs.dump(projectDir), 'ui5-local.yaml')).toBe(false);
```

Comment:
Do we really need a dump here? What about `fs.existsInMemory(filepath)`?

Reply:
I used `fs.exists()` instead .. is that alright?